src/toil/provisioners/abstractProvisioner.py

```diff
@@ -118,6 +118,10 @@ def __init__(self, clusterName=None, zone=None, nodeStorage=50):
         self.clusterName = clusterName
         self._zone = zone
         self._nodeStorage = nodeStorage
+        self._nodeStorageOverrides = {}
+        for override in nodeStorageOverrides or []:
+            nodeShape, storageOverride = override.split(':')
+            self._nodeStorageOverrides[nodeShape] = int(storageOverride)
```

Comment:
Since the override string is coming straight from the user, we might want to have some error handling here so we report a badly written entry instead of just crashing with a stacktrace.

Reply:
That's a good point! I decided to add it as a `checkFn` in common.py as it seems like most arg error checking is done there anyway.